src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -171,4 +171,8 @@ public static ScheduleTypes fromName(String name) {
         public static final String TRANSACTION_DESCRIPTION_STARTER = "AWS CHARGES ";
     }
 
+    public static class AccountFundsUpdateConstants {
+        public final static Integer MAX_TOTAL_ALLOWED = 10000;
```

Comment:
It was not part of the requirements, but just an idea - this is a good candidate for a system parameter as the functionals may change their mind about the max allowed value and it is easier to control that using a system parameter